tests/__init__.py

```diff
@@ -0,0 +1,11 @@
+import os
```

Comment:
@cdkini @anthonyburdi @billdirks @Kilo59 -- Something tells me that OSS should not have any mentions of `GE_CLOUD` or other non-OSS type branding (similar sentiment as not including internal JIRA ticket links in the OSS code), but wanted to know what you think.  Thanks. /cc @Shinnnyshinshin